### DISCUSSION TITLE
`Route.push` not redirecting to another page (maybe due to next.config.js)

### DISCUSSION THREAD

**Discussion by fillipo** [vercel/next.js#14658] [Help]
# Bug report
Perhaps current next.config.js config seems to prevent the `Route.push` to redirecting the page. If you try to redirect nothing happens. Not sure if it is something the way less/css is built.

## To Reproduce

1. Create a test page with the following code

```
import Router from 'next/router';

export default function TestRoute() {return(<button type="button" onClick={() => Router.push('/')}>TEST</button>)}
```

2. npm run dev - go go the test page and click on button

## Expected behavior
Button click should redirect to main home index page

## System information

- OS: [e.g. macOS]

```
 "dependencies": {
    "@zeit/next-less": "^1.0.1",
    "antd": "^3.5.4",
    "babel-plugin-import": "^1.7.0",
    "less": "3.0.4",
    "less-vars-to-js": "1.3.0",
    "next": "latest",
    "null-loader": "2.0.0",
    "react": "^16.7.0",
    "react-dom": "^16.7.0"
  },
```

## Pontential solution
This setup worked and fixed the issue for me: https://medium.com/@tanakorn0412/customize-ant-design-theme-in-next-js-version-9-5958fbc531a3

**Comment by jamesmosier** [comment]
Hi @fillipo. Do you have a more complete reproduction that causes this? I wasn't able to reproduce with the steps you outlined.
